runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -331,8 +329,10 @@ public void onGameTick(GameTick event)
 
 		if (client.getGameState() != GameState.LOGGED_IN
 			|| local == null
-			|| System.currentTimeMillis() - client.getMouseLastPressedMillis() < 200
-			|| client.getKeyboardIdleTicks() < 10)
+			// If user has interacted with game in last second then they're not idle so don't send idle notification
+			|| System.currentTimeMillis() - client.getMouseLastPressedMillis() < 1000
```

Comment:
Is it necessary to change this and the keyboard timeout?

Reply:
The reason I changed it is that they were both too short before to make any difference about 2/3rds of the time. Game ticks are every 600ms (and it seems the onGameTick method actually only runs every 600-800ms unless debug mode slows it down) so most of the time it would notify you even if you were the one to click away from the fishing spot.

Although keyboard isn’t really used while fishing it made sense that it would have the same issue as mouse clicks so I did both.

Reply:
I've put the keyboard timeout and mouse idle ticks timeout back (along with the unit test) and tidied up the commit history.